src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail.html

```diff
@@ -40,7 +42,9 @@ <h1 flex class="md-title kd-replicationcontrollerdetail-app-name">
       <div flex layout="column" class="kd-replicationcontrollerdetail-sidebar-info">
         <span class="kd-replicationcontrollerdetail-sidebar-line">Name</span>
         <span class="kd-replicationcontrollerdetail-sidebar-subline">
-          {{::ctrl.replicationControllerDetail.name}}
+          <kd-middle-ellipsis display-string="{{::ctrl.replicationControllerDetail.name}}"
+                              max-length="{{::ctrl.nameMaxLength}}">
```

Comment:
Hmm... I see plenty of free space left here. Should we use different max length?